lib/srv/discovery/common/database.go

```diff
@@ -1290,7 +1290,7 @@ func ExtraElastiCacheLabels(cluster *ectypes.ReplicationGroup, tags []ectypes.Ta
 
 // ExtraMemoryDBLabels returns a list of extra labels for provided MemoryDB
 // cluster.
-func ExtraMemoryDBLabels(cluster *memorydb.Cluster, tags []*memorydb.Tag, allSubnetGroups []*memorydb.SubnetGroup) map[string]string {
+func ExtraMemoryDBLabels(cluster *memorydbtypes.Cluster, tags []memorydbtypes.Tag, allSubnetGroups []memorydbtypes.SubnetGroup) map[string]string {
```

Comment:
since you are here, do you mind adding the engine label similar to the elasticache? 🙏